helpers/lib/cdnify.js

```diff
@@ -26,28 +24,38 @@ module.exports = globals => {
             return '';
         }
 
-        if (/^(?:https?:)?\/\//.test(path)) {
+        if (path.startsWith('https://') || path.startsWith('http://') || path.startsWith('//')) {
             return path;
         }
-
+        // This block is to strip leading . and / from path
         if (['.', '/'].includes(path[0])) {
-            path = path.replace(/^[\.\/]+/, '');
+            for (let i = 0; i < path.length; i++) {
+                if (path[i] !== '.' && path[i] !== '/') {
+                    path = path.slice(i);
+                    break;
+                }
+            }
         }
 
-        if (protocolMatch.test(path)) {
-            var match = path.match(protocolMatch);
-            path = path.slice(match[0].length, path.length);
+        let colonIndex = path.indexOf(':');
+
+        if (colonIndex !== -1) {
+            while (path[colonIndex + 1] === ':') {
```

Comment:
You probably need an additional guard here to avoid out of bounds errors: `while (colonIndex + 1 < path.length && path[colonIndex + 1] === ':')`

Reply:
with Javascript there wouldn't be an out of bounds error just return undefined.  Which would break the loop.  If you then slice the string it would be an empty string.

Reply:
Just realized we wouldn't actually slice the string if that were the case.

Reply:
Err.  Just realized I thought we were thinking of some other piece of code instead of this one.